src/views/AddPriceMultiple.vue

```diff
@@ -434,6 +439,41 @@ export default {
       if (this.recentLocations.length) {
         this.setLocationData(this.recentLocations[0])
       }
+      // get a list of categories to init the autocomplete - might be replaced with hardcoded categories
+      const initVal = 'a'
+      this.queryCategory(initVal)
+    },
+    queryCategory(newVal) {
+      this.loading = true
+      if (newVal && newVal.length > 0) {
+        this.category_loading = true
+        api
+          .getCategories(newVal, 'category', 10)
```

Comment:
on every new price(s) we fetch the categories ? this isn't ideal :sweat_smile: 
- we could fetch them when the language is changed in the user settings
- we could have them directly in the raw json

Reply:
maybe document some implementation solutions first in the corresponding issue ? #224

Reply:
Here the API is called:
- once at page mount (with a simple query "a")
- Everytime the user inputs something in autocomplete field. So for example if the user type "rad", it will call for "r", then "ra" and "rad"

The fetch returns a list of max 10 categories (max number can be changed) and this list will be shown in the autocomplete field.

I will list solutions in the issue